anvil/src/main/java/trikita/anvil/Anvil.java

```diff
@@ -128,9 +120,11 @@ private static Runnable createRenderTask() {
         return new Runnable() {
             @Override
             public void run() {
-                Set<Mount> set = new HashSet<>(mounts.values());
-                for (Mount m : set) {
-                    render(m);
+                synchronized (mounts) {
```

Comment:
I think this synchronized is excessive since mounts are already use synchronizedMap